docs/docs/guide/debugging_table/table.tsx

```diff
@@ -0,0 +1,342 @@
+                location.href = '#Details';
+              }}>
+              ✅*
+            </button>
+          </td>
+          <td style={{ textAlign: 'center' }}>
+            <button
+              onClick={() => {
+                setDetails('Safari DevTools/hermes/iOS');
+                location.href = '#Details';
+              }}>
+              N/A
+            </button>
+          </td>
+          <td style={{ textAlign: 'center' }}>
+            <button
+              onClick={() => {
+                setDetails('Safari DevTools/V8/iOS');
+                location.href = '#Details';
+              }}>
+              N/A
+            </button>
+          </td>
+        </tr>
+        <tr>
+          <td rowSpan={2}>React DevTools</td>
+          <td style={{ textAlign: 'center' }}>Android</td>
+          <td style={{ textAlign: 'center' }}>
+            <button
+              onClick={() => {
+                setDetails('React DevTools/JSC/Android');
+                location.href = '#Details';
+              }}>
+              ✅
+            </button>
+          </td>
+          <td style={{ textAlign: 'center' }}>
+            <button
+              onClick={() => {
+                setDetails('React DevTools/hermes/Android');
+                location.href = '#Details';
+              }}>
+              ✅
+            </button>
+          </td>
+          <td style={{ textAlign: 'center' }}>
+            <button
+              onClick={() => {
+                setDetails('React DevTools/V8/Android');
+                location.href = '#Details';
+              }}>
+              ✅
+            </button>
+          </td>
+        </tr>
+        <tr>
+          <td style={{ textAlign: 'center' }}>iOS</td>
+          <td style={{ textAlign: 'center' }}>
+            <button
+              onClick={() => {
+                setDetails('React DevTools/JSC/iOS');
+                location.href = '#Details';
+              }}>
+              ✅
+            </button>
+          </td>
+          <td style={{ textAlign: 'center' }}>
+            <button
+              onClick={() => {
+                setDetails('React DevTools/hermes/iOS');
+                location.href = '#Details';
+              }}>
+              ✅
+            </button>
+          </td>
+          <td style={{ textAlign: 'center' }}>
+            <button
+              onClick={() => {
+                setDetails('React DevTools/V8/iOS');
+                location.href = '#Details';
+              }}>
+              N/A
+            </button>
+          </td>
+        </tr>
+      </table>
+
+      <h4>Key to symbols:</h4>
+      <ul>
+        <li>
+          ✅ - available in apps using Reanimated, an asterisk indicates that
+          some limitations apply
+        </li>
+        <li>❌ - unavailable in apps using Reanimated</li>
+        <li>N/A - unavailable in React Native apps</li>
+      </ul>
+
+      <h3 style={{ marginBottom: 0 }} id="Details">
```

Comment:
Looks like Docusaurus uses `kebab-case` for header IDs, e.g. https://docs.swmansion.com/react-native-reanimated/docs/guide/testing#recommended-testing-library
```suggestion
      <h3 style={{ marginBottom: 0 }} id="details">
```